apps/sptribs/automation/kustomization.yaml

```diff
@@ -6,5 +6,9 @@ resources:
   - ../sptribs-case-api/ithc-image-policy.yaml
   - ../sptribs-case-api/perftest-image-policy.yaml
   - ../sptribs-case-api/demo-image-policy.yaml
+  - ../sptribs-dss-update-case-web/image-policy.yaml
+  - ../sptribs-dss-update-case-web/demo-image-policy.yaml
+  - ../sptribs-dss-update-case-web/ithc-image-policy.yaml
+  - ../sptribs-dss-update-case-web/perftest-image-policy.yaml
```

Comment:
Do we also need a reference to `../sptribs-dss-update-case-web/ithc-image-repo.yaml`?

Reply:
We don't have an ithc-image-repo.yaml but have added a reference to our ../sptribs-dss-update-case-web/image-repo.yaml in latest commit